src/controller/java/src/chip/devicecontroller/model/EventState.java

```diff
@@ -36,12 +39,14 @@ public final class EventState {
   public EventState(
       long eventNumber,
       int priorityLevel,
+      int timestampType,
       long systemTimeStamp,
```

Comment:
Could you also rename systemTimestamp to timestampValue?

Reply:
Yes,  I also noticed this name is not appropriate, but this change will break backward compatibility since this is part of published API.